tensorflow/core/platform/platform.h

```diff
@@ -51,13 +51,6 @@ limitations under the License.
 // If no platform specified, use:
```

Comment:
Looks like we can just remove this entire #elif branch, as it's identical the #else branch.

Reply:
Ah, yeah. Thanks, will update